python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1146
   - name: u_calibFlux
     "@id": "#Object.u_calibFlux"
     datatype: double
     description: Flux within 12.0-pixel aperture. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1147
   - name: u_calibFluxErr
     "@id": "#Object.u_calibFluxErr"
     datatype: double
     description: Flux uncertainty within 12.0-pixel aperture. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1148
   - name: u_calibFlux_flag
     "@id": "#Object.u_calibFlux_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1149
   - name: u_calibFlux_flag_apertureTruncated
     "@id": "#Object.u_calibFlux_flag_apertureTruncated"
     datatype: boolean
     description: Aperture did not fit within measurement image. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1150
   - name: u_calibFlux_flag_sincCoeffsTruncated
     "@id": "#Object.u_calibFlux_flag_sincCoeffsTruncated"
     datatype: boolean
     description: Full sinc coefficient image did not fit within measurement image. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1151
   - name: u_calib_astrometry_used
     "@id": "#Object.u_calib_astrometry_used"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1152
   - name: u_calib_photometry_reserved
     "@id": "#Object.u_calib_photometry_reserved"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1153
   - name: u_calib_photometry_used
     "@id": "#Object.u_calib_photometry_used"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1154
   - name: u_calib_psf_candidate
     "@id": "#Object.u_calib_psf_candidate"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1155
   - name: u_calib_psf_reserved
     "@id": "#Object.u_calib_psf_reserved"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1156
   - name: u_calib_psf_used
     "@id": "#Object.u_calib_psf_used"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1157
   - name: u_centroid_flag
     "@id": "#Object.u_centroid_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1158
```

Comment:
These per-band pixel-unit centroid columns should go right after the per-band ra/dec columns, since they're just WCS transforms of each other, and the flag applies to the ra/dec as well as the pixel-valued centroids.